src/lerna_manifest.rs

```diff
@@ -17,21 +17,25 @@ struct LernaManifestFile {
     packages: Vec<String>,
```

Comment:
(attaching comment to arbitrary line)

This file name is no longer accurate now that it supports npm.

Reply:
Fixed in 29a8999